common/app/views/fragments/omnitureScript.scala.html

```diff
@@ -1,21 +1,12 @@
 @(item: Option[model.MetaData])
 
 @import conf.Static
-@import conf.Configuration
 @import views.support.OmnitureAnalyticsAccount
 
 @item.map{ item =>
     <script>
         @* must always be set before the Omniture file is parsed *@
         window.s_account = '@OmnitureAnalyticsAccount(item)';
+        @Html(Static.js.omnitureJs)
     </script>
 }
-
-<script>
```

Comment:
have you tested this works? is it possible `s_account` could be referenced before the assignment?